### ISSUE TITLE
The solution code doesn't pass the check!

### ISSUE THREAD

**Issue by akashvevo** [FreeCodeCamp/FreeCodeCamp#10952]

**Comment by erictleung** [comment]
@akashvevo can you give more information about which challenge and what solution code that doesn't pass? We need more information to be able to help with your issue. Thanks!

**Comment by kattbab1** [comment]
I'm not sure what this venue is about to can someone please break it down in layman's terms I'm a computer retard give me some help

**Comment by atjonathan** [comment]
Closing as stale.
